src/main/java/school/hei/haapi/service/StudentFileService.java

```diff
@@ -28,16 +32,24 @@ public byte[] generatePdf(String studentId, String template) {
   }
 
   private Context loadContext(String studentId) {
+    ClassPathResourceResolver pathResolver = new ClassPathResourceResolver();
```

Comment:
Also a suitable variable name would be "resourceResolver" or "classPathResourceResolver'
pathResolver induces that you only resolve the path and that you get a path